addon/ng2/utilities/route-utils.ts

```diff
@@ -70,6 +70,9 @@ export function bootstrapItem(mainFile, imports: {[key: string]: [string, boolea
 
 export function insertImport(fileToEdit: string, symbolName: string,
                                   fileName: string, isDefault = false): Change {
+        
+  fileName = fileName.replace(/\\/, '/'); // correction in windows           
```

Comment:
hi, hansl.
I tried to use path.posix.normalize() . but the result wasn't correct. the Result is shown as below.

before normalize. fileName is './aa\bb/bb.component'
after normalize fileName is 'aa\bb/bb.component'

I couldn't find the reason why this didn't work. my node version is 6.3.1. 

A little more help, please. Thanks